packages/fast-layouts-react/src/pane/pane.tsx

```diff
@@ -145,6 +146,7 @@ export class Pane extends Foundation<PaneHandledProps, PaneUnhandledProps, PaneS
             width: this.props.initialWidth,
         };
 
+        this.onResize = throttle(this.onResize, 16);
```

Comment:
Should we be throttling callbacks or letting authors do it at the other end?  My thought would be that we should send out a resize event in precise synch with our own calls to setState to update the internal size of the pane so that an author's changes can (theoretically) take effect on the same render pass.  So either not throttle at all, or if we throttle we do so such that our internal calls that set the width in state are in the same throttled code that sends out the event?